contrib/prometheus/prometheus-all.json

```diff
@@ -21,14 +21,14 @@
             "spec": {
                 "containers": [
                     {
-                        "name":"kube-promdash",
-                        "image": "prom/promdash",
                         "env": [
                             {
                                 "name": "DATABASE_URL",
                                 "value": "sqlite3:/promdash/file.sqlite3"
                             }
                         ],
+                        "image": "prom/promdash",
+                        "name": "kube-promdash",
```

Comment:
Why did you move that - I think having name and image at the beginning is more intuitive.